client/my-sites/plan-features/footer.jsx

```diff
@@ -4,25 +4,71 @@
 import { localize } from 'i18n-calypso';
 import noop from 'lodash/noop';
 import React, { PropTypes } from 'react';
+import filter from 'lodash/filter';
+import reduce from 'lodash/reduce';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import Gridicon from 'components/gridicon';
+import classNames from 'classnames';
+import { plansList } from 'lib/plans/constants';
 
-const PlanFeaturesFooter = ( { translate, current = false, available = true, description, onUpgradeClick = noop } ) => {
+function getLongestDescription( planName ) {
+	if ( ! planName ) {
+		return null;
+	}
+	const isJetpack = planName.indexOf( 'jetpack' ) === 0;
+	const plans = Object.keys( plansList );
+
+	return reduce(
+		filter( plans, ( plan ) => {
+			const isPlanJetpack = plan.indexOf( 'jetpack' ) === 0;
```

Comment:
> What's the difference between isJetpack and isPlanJetpack? I would name it as follows

I'm basically terrible at naming things. I'm filtering for wpcom plans or jetpack plans depending on what the current plan is. I'll take another stab at it.

re: the reduce and filter, we've been burned by a lot of issues regarding what is and isn't polyfilled, so I figured it was easier to just use lodash methods to avoid ambiguity.